libs/core/src/lib/card/kpi/card-kpi-header.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, ChangeDetectionStrategy, ElementRef, Renderer2 } from '@angular/core';
+
+import { CLASS_NAME } from '../constants';
+
+@Component({
+    selector: 'fd-card-kpi-header',
+    templateUrl: './card-kpi-header.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CardKpiHeaderComponent implements OnInit {
+    /** @hidden */
+    constructor(private _elementRef: ElementRef<HTMLElement>, private _renderer: Renderer2) {}
+
+    /** @hidden */
+    ngOnInit(): void {
+        this._addClassNameToHostElement(CLASS_NAME.cardAnalyticalArea);
+    }
+
+    /**@hidden */
+    private _addClassNameToHostElement(className: string): void {
+        this._addClassName(this._elementRef.nativeElement, className);
+    }
+
+    /**@hidden */
+    private _addClassName(element: HTMLElement, className: string): void {
+        this._renderer.addClass(element, className);
+    }
```

Comment:
Do we need this extra function? Can we just call `this._renderer.addClass(element, className)` directly in `_addClassNameToHostElement` as is done in other components?

Reply:
Changed it so now it implements cssClassBuilder interface since it was requested by other reviews